ecs-agent/tmds/handlers/fault/v1/handlers/handlers_test.go

```diff
@@ -38,10 +39,40 @@ import (
 )
 
 const (
-	endpointId  = "endpointId"
-	port        = 1234
-	protocol    = "tcp"
-	trafficType = "ingress"
+	endpointId         = "endpointId"
+	port               = 1234
+	protocol           = "tcp"
+	trafficType        = "ingress"
+	taskARN            = "taskArn"
+	awsvpcNetworkMode  = "awsvpc"
+	deviceName         = "eni-123"
```

Comment:
nit: it usually looks like `eth0`/`eth1`/...

Reply:
ah oops good catch.